src/Parser/DeathKnight/Frost/Modules/Features/AlwaysBeCasting.js

```diff
@@ -42,16 +42,26 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     */
   ];
 
-  suggestions(when) {
-    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+  get downtimeSuggestionThresholds() {
+    return {
+      actual: this.downtimePercentage,
+      isGreaterThan: {
+        minor: 0.2,
+        average: 0.35,
+        major: .4,
+      },
+      style: 'percentage',
+    };
+  }
 
-    when(deadTimePercentage).isGreaterThan(0.2)
+  suggestions(when) {
+    when(this.downtimeSuggestionThresholds.actual).isGreaterThan(this.downtimeSuggestionThresholds.isGreaterThan.minor)
       .addSuggestion((suggest, actual, recommended) => {
```

Comment:
You can remove the entire ``isGreaterThan`` bit

Reply:
Oh that's much cleaner.  I updated Unholy's downtime suggestion too since I copied this from there.